bokeh/tests/test_objects.py

```diff
@@ -181,6 +181,16 @@ def test_to_json(self):
                          (child_obj._id, obj._id),
                          json_string)
 
+    def test_no_units_in_json(self):
+        from bokeh.models import AnnularWedge
+        self.maxDiff = None
+        obj = AnnularWedge()
+        json = obj.to_json()
+        self.assertTrue('start_angle' in json)
+        self.assertTrue('start_angle_units' not in json)
+        self.assertTrue('outer_radius' in json)
+        self.assertTrue('outer_radius_units' not in json)
```

Comment:
I'm working on that now. I have a follow-on PR that will make it a lot easier to make _units props truly 'virtual'.
Hmm. This may actually remove the need to put serialized= kwarg on every constructor, come to think of it.